data/lavezzo2020suppression/lavezzo2020suppression-extraction.md

```diff
@@ -0,0 +1,250 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.4
+  kernelspec:
+    display_name: python_clean
+    language: python
+    name: python3
+---
+
+```python
+import yaml
+import pandas as pd
+import numpy as np
+import textwrap
+import datetime
+
+
+#functions to add folded blocks and literal blocks;
+class folded_str(str): pass
+class literal_str(str): pass
+
+def folded_str_representer(dumper, data):
+    return dumper.represent_scalar(u'tag:yaml.org,2002:str', data, style='>')
+def literal_str_representer(dumper, data):
+    return dumper.represent_scalar(u'tag:yaml.org,2002:str', data, style='|')
+
+yaml.add_representer(folded_str, folded_str_representer)
+yaml.add_representer(literal_str, literal_str_representer);
+```
+
+```python
+#Read Excel file into the environment
+df_demo = pd.read_excel('anonymised_data_public_final.xlsx', sheet_name='anonymised_dataset')
```

Comment:
Maybe we can name this `df_demographics`? It took me a little bit to figure out what the "demo" meant.

Reply:
Agree. It would be better to give more informative names to those dataframes.

Reply:
Fixed